.github/workflows/build_n_deploy_dev.yaml

```diff
@@ -24,8 +24,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
         run: |
-          yarn
+          yarn install
           yarn build
+          yarn install --production 
```

Comment:
Dette gjør at vi fjerner devDependenciene. Vi må kjøre `yarn install` førs fordi vi trenger det for å kjøre `yarn build`